recipes/cad1/task1/baseline/README.md

```diff
@@ -0,0 +1,148 @@
+└───task1
+    └───audio
+        └───MedleyDB
+            ├───Audio
+            └───Metadata
+```
+
+* **BACH10** contains the BACH10 dataset [6].
+BACH10 dataset is not used in the evaluation.
+```text
+cadenza_data
+└───task1
+    └───audio
+        └───fma_small
+            ├───000
+            ├───001
+            ├───...
+```
+
+* **FMA Small** contains the FMA small subset of the FMA dataset [7].
+FMA small dataset is not used in the evaluation. This dataset does not provide the independent stems, but only the full mix.
+However, it can be used for any unsupervised learning task.
+```text
+cadenza_data
+└───task1
+    └───audio
+        └───fma_small
+            ├───000
+            ├───001
+            ├───...
+```
+
+## 2. Baseline
+
+In the `baseline/` folder, we provide code for running the baseline enhancement system and performing the objective evaluation.
+Note that we use hydra for config handling. For more information, see [here](https://hydra.cc/docs/intro/).
+
+### 2.1 Enhancement
+
+The baseline enhance simply takes the out-of-the-box [Hybrid Demucs](https://github.com/facebookresearch/demucs)[1]
+source separation model distributed on [TorchAudio](https://pytorch.org/audio/main/tutorials/hybrid_demucs_tutorial.html)
+and applies a simple NAL-R [2] fitting amplification and a simple automatic gain compressor to each
+VDBO (`vocals`, `drums`, `bass` and `others`) stem.
+
+The remixing is performed by summing the amplified VDBO stems.
+
+The baseline will generate a left and right signal for each VDBO stem and a remixed signal, totalling 9 signals per song-listener.
+
+To run the baseline enhancement system, firstly specify `paths.root` in `config.yaml` to point to
+where you have installed the clarity data. You can also define your own `path.exp_folder` to store enhanced
+signals and evaluated results.
+Then run:
+```bash
+python enhance.py
+```
+Alternatively, you can provide the root variable on the command line, e.g.,
+```bash
+python enhance.py path.root=/full/path/to/my/cadenza_data
+```
+To get a full list of the parameters, run:
+```bash
+python enhance.py --help
+```
+
+The folder `enhanced_signals` will appear in the `exp` folder.
+
+### 2.2 Evaluation
+
+The `evaluate.py` simply takes the signals stored in `enhanced_signals` and computes the HAAQI [3] score
+for each of the eight left and right VDBO stems.
+The average of these eight scores is computed and returned for each signal.
+
+To run the evaluation stage, make sure that `path.root` is set in the config.yaml file and then run
+```bash
+python evaluate.py
+```
+A csv file containing the eight HAAQI scores and the combined score will be generated in the `path.exp_folder`.
+
+To check the HAAQI code, see [here](../../clarity/evaluator/haaqi).
+
+Please note: you will not get identical HAAQI scores for the same signals if the random seed is not determined
+(in the given recipe, the random seed for each signal is set the last eight digits of the song md5).
```

Comment:
```suggestion
(in the given recipe, the random seed for each signal is set as the last eight digits of the song md5).
```

Reply:
Unresolving because whether `as` or `to` is inserted currently its not grammatically correct to have `_is set the last eight digits of the song md5`.